client/webserver/site/src/css/market.scss

```diff
@@ -427,6 +427,12 @@ table.balance-table button:hover {
   }
 }
 
+div.numorders {
+  color: white;
```

Comment:
No white text, please. Inherited `color` will work fine once you get the background colors adjusted.